### PR TITLE
Add Copy as cURL links for Cloud and ECE

### PR DESCRIPTION
**Pull request by nik9000**
Adds "Copy as cURL" links for Elastic Cloud SAAS and Elastic Cloud
Enterprise.

Also removes the "langStrings" passed into the curl building code. We
don't need it.

### CODE REVIEW

**Comment by nik9000** [comment]
Thanks for reviewing @JoshMock! I'm going to wait on some folks from cloud to have a look too.

File: resources/web/docs_js/index.js [hunk 1]
```diff
@@ -194,6 +206,32 @@ $(function() {
...
   init_console_widgets();
   init_kibana_widgets();
+  $("div.ess_widget").each(function() {
+    const div         = $(this),
+          snippet     = div.attr('data-snippet'),
+          consoleText = div.prev().text() + '\n';
+
+    return mount(div, ConsoleWidget, {
+      setting: "ess",
+      url_label: 'Enter the URL of Elastic Cloud',
```
Comment (nrichers) [inline resources/web/docs_js/index.js]: I would change to `Enter the endpoint URL of the Elasticsearch Service`.

Rationale: 
* I made a global comment earlier about making sure we use the right service name ... just to repeat that comment here: we definitely want to use `Elasticsearch Service`, as the Elastic Cloud umbrella name now encompasses all the SaaS offerings we have (including things like Elastic App Search Service). 
* Also, I would change `URL` --> `endpoint URL`, as that is the terminology we use in the ESS UI.

File: resources/web/docs_js/index.js [hunk 2]
```diff
@@ -194,6 +206,32 @@ $(function() {
...
+      configure_text: 'Configure Elastic Cloud URL',
```
Comment (nrichers) [inline resources/web/docs_js/index.js]: Similar change here.

File: resources/web/docs_js/index.js [hunk 3]
```diff
@@ -194,6 +206,32 @@ $(function() {
...
+    });
+  });
+  $("div.ece_widget").each(function() {
+    const div         = $(this),
+          snippet     = div.attr('data-snippet'),
+          consoleText = div.prev().text() + '\n';
+
+    return mount(div, ConsoleWidget, {
+      setting: "ece",
+      url_label: 'Enter the URL of Elastic Cloud',
```
Comment (nrichers) [inline resources/web/docs_js/index.js]: And for lines 229 and 230 (refer to my comment at line 216).

File: resources/web/docs_js/localization.js [hunk 4]
```diff
@@ -8,6 +8,12 @@ export const lang_spec = {
   "Configure Kibana URL": {
     "zh_cn": "配置 Kibana URL"
   },
+  "Configure Elastic Cloud URL": {
+    "zh_cn": "配置 Elastic Cloud URL"
+  },
+  "Configure Elastic Cloud Enterprise URL": {
+    "zh_cn": "配置 Elastic Cloud Enerprise URL"
```
Comment (nrichers) [inline resources/web/docs_js/localization.js]: `配置 Elastic Cloud Enterprise endpoint URL` (note the `Enerprise` typo)

File: resources/web/docs_js/localization.js [hunk 5]
```diff
@@ -8,6 +8,12 @@ export const lang_spec = {
   "Configure Kibana URL": {
     "zh_cn": "配置 Kibana URL"
   },
+  "Configure Elastic Cloud URL": {
+    "zh_cn": "配置 Elastic Cloud URL"
```
Comment (nrichers) [inline resources/web/docs_js/localization.js]: `配置 Elasticsearch Service endpoint URL`

**Comment by nik9000** [comment]
@nrichers, how does sit look now?

File: README.asciidoc [hunk 6]
```diff
@@ -1050,13 +1050,23 @@ footnote to a particular line of code:
...
-Code blocks can be followed by a "View in Console" link which, when clicked,
-will open the code snippet in Console. This is how you do it:
+Code blocks can be followed by a "Copy as cURL" link which will convert the
+snippet into a sequence of calls to the ubiquitous
+https://curl.haxx.se/[cURL] tool that work in the bash shell and copy it to
+the clipboard. Similarly, if the target of the snippet is Elasticsearch we also
+add a "View in Console" link will open the code snippet in Console.
 
-.Code block with CONSOLE link
+You enable the it be setting the "language" of the snippet to a supported
```
Comment (kilfoyle) [inline README.asciidoc]: "the it be" -> "the console link by"

**Comment by nik9000** [comment]
Thanks for reviewing @kilfoyle! I've pushed a fix for the typo and I'll merge what that is in.

**Pull request merged**